examples/with-particles/package.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "with-particles",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "next": "12.1.0",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
+    "react-tsparticles": "^1.42.4"
+  },
+  "devDependencies": {
+    "eslint": "8.11.0",
+    "eslint-config-next": "12.1.0"
```

```suggestion
    "next": "latest",
    "react": "18.1.0",
    "react-dom": "18.1.0",
    "react-tsparticles": "^1.42.4"
  },
  "devDependencies": {
    "eslint": "8.11.0",
    "eslint-config-next": "latest"
```

Let's make sure dependencies are up-to-date as of writing.